Modules/TemperatureSensor/TemperatureSensor.csproj

```diff
@@ -13,8 +13,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MathNet.Numerics" Version="4.5.1" />
```

Comment:
please make sure that these packages are dotnet standard libraries. In other words, the app can run in Linux